org.ioquake3.IOQuake3.yaml

```diff
@@ -1,35 +1,47 @@
 ---
 app-id: org.ioquake3.IOQuake3
 runtime: org.freedesktop.Platform
-runtime-version: '20.08'
+runtime-version: '22.08'
 sdk: org.freedesktop.Sdk
 branch: stable
```

Comment:
I have removed the branch from all my Flatpaks because it caused weird issues and was not needed on Flathub.